From 4f3a2f0fee6ccec8e50682c2d05c23333ec99ab6 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 21 May 2026 10:16:41 +0100
Subject: [PATCH] common: update submodule

  Richard W.M. Jones (1):
        mlcustomize: Drop setfiles -F flag

(cherry picked from commit 85c3a1989733879381456d71d716619160539d5b)
---
 common | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Submodule common a901c174..9584c82f:
diff --git a/common/mlcustomize/SELinux_relabel.ml b/common/mlcustomize/SELinux_relabel.ml
index f1729e3f..62450e41 100644
--- a/common/mlcustomize/SELinux_relabel.ml
+++ b/common/mlcustomize/SELinux_relabel.ml
@@ -122,4 +122,4 @@ and use_setfiles g =
             Array.of_list in
 
   (* Relabel everything. *)
-  g#setfiles ~force:true specfile mps
+  g#setfiles specfile mps
